README.md

```diff
@@ -32,7 +32,6 @@ To install ProtoData on a *nix system:
  ```
  ./target/dir/install.sh $HOME/custom/installation/dir
  ```
- In any case, the directory must exist. Otherwise, `install.sh` will exit with an error.
 
  4. If you use either `bash` or `zsh` shell, you will be asked to run a `source` command to update
```

Comment:
I thought this file would reflect the support of multiple source sets. I would want to tell end-users how to use ProtoData in their tests, for instance.

Reply:
Added a description of using ProtoData on main/test scopes.